research/object_detection/utils/ops.py

```diff
@@ -32,8 +32,12 @@
 from object_detection.utils import static_shape
 
 
+multilevel_matmul_crop_and_resize = \
```

Comment:
No need to add these symbols here. Directly import spatial_ops in model_builder.py and use these functions.